app/views/layouts/application.html.haml

```diff
@@ -12,3 +12,6 @@
         = yield
 
     = render 'shared/footer'
+
+  :javascript
+  var navigation = $('#nav-main').okayNav();
```

Comment:
Also name the `#nav-menu` to be something like `#category-menu` which is more descriptive.